pyexchange/dydx.py

```diff
@@ -151,7 +151,7 @@ def place_order(self, pair: str, is_sell: bool, price: float, amount: float) ->
         created_order = self.client.place_order(
             market=pair,  # structured as <MAJOR>-<Minor>
             side=side,
-            price=Decimal(price),
+            price=round(Decimal(price), 2),
```

Comment:
is this something we can retrieve from DyDx API / python client?

Reply:
Yep, code has been added to retrieve that and `market_id` from the new `get_markets()` endpoint